manifests/o/OpenWhisperSystems/Signal-Beta/5.2.0-beta.1/OpenWhisperSystems.Signal-Beta.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: OpenWhisperSystems.Signal-Beta
```

```suggestion
PackageIdentifier: OpenWhisperSystems.Signal.Beta
```
Lets do this so it looks cleaner (this is a new feature which is why older manifests use `-`